planck-cljs/src/planck/io.cljs

```diff
@@ -346,12 +346,67 @@
 (defn ^boolean directory?
   "Checks if dir is a directory."
   [dir]
-  (js/PLANCK_IS_DIRECTORY (:path (as-file dir))))
+  (boolean (js/PLANCK_IS_DIRECTORY (:path (as-file dir)))))
 
 (s/fdef directory?
   :args (s/cat :dir (s/or :string string? :file file?))
   :ret boolean?)
 
+(defn exists?
+  "Checks if f exists on disk."
+  [f]
+  (not (nil? (file-attributes f))))
+
+(s/fdef exists?
+  :args (s/cat :f (s/or :string string? :file file?))
+  :ret boolean?)
+
+(defn path-elements
+  "Returns the path elements of x as a sequence."
+  [x]
+  (if x
+    (remove (partial = "") (string/split (str x) #"/"))))
```

Comment:
Hah. Using `str` is clever. In other places `(:path (as-file x))` is used.

Reply:
Let me know if you want me to change it!

Reply:
I think it is fine as is. I'd only change it for consistency with other code.

Reply:
Fixed in rev 27f75c0.